shaders/src/light_indirect.fs

```diff
@@ -676,14 +671,26 @@ void applyRefraction(
     float lod = max(0.0, 2.0 * log2(tweakedPerceptualRoughness) + frameUniforms.refractionLodOffset);
 
     vec4 Fat = textureLod(light_ssr, p.xy, lod);
-    float at = Fat.a;
-#if defined(IS_TRANSPARENT_OR_REFRACTIVE)
-    vec3 Ft = mix(vec3(1.0), Fat.rgb, at);
-#else
-    vec3 Ft = Fat.rgb;
 #endif
 
-#endif
+    vec3 Ft;
+    float at;
+    if (frameUniforms.needsAlphaChannel == 1.0) {
+        // Bias transparent view using the following formula:
+        // https://twitter.com/garrettkjohnson/status/1526645080525705216
+        const vec4 backgroundColor = vec4(1.0);
+        const float opacity = 0.4;
```

Comment:
Why is `opacity` fixed at `0.4`?

Reply:
`opacity` denotes how much we want to bias towards the forground color. Here you can see what happens when you increase `opacity` from `0.4` to `0.5` all the way to `0.6`.
<img width="446" alt="Screenshot 2022-11-30 at 13 53 10" src="https://user-images.githubusercontent.com/6261494/204801286-6a327df2-03cb-425b-9bd1-44bd4fb2d138.png">

Reply:
I think `0.4` works better on both light and dark background, but maybe we should ask @alex-marton.

It maybe could be a `#define` somewhere instead of just simply a value burnt in here, wdyt?

Reply:
I don't have a string opinion on whether it should be a `#define` or not. Either way is fine by me.
Asking Alex is a good idea though.